nginx/conf.d/php_backend.conf

```diff
@@ -0,0 +1,12 @@
+## php backend settings
+fastcgi_pass    127.0.0.1:9000; #swap :9000 with :$port_switch when configuring hhvm/php-fpm routes;
+fastcgi_index   index.php;
+include         fastcgi_params;
+
+## specific security and compatibility headers
+add_header X-Frame-Options SAMEORIGIN;
```

Comment:
maybe is better to send headers like these from the server not from the application.